packages/blockchain-extension/resources/tutorials/developer-tutorials/privateData.md

```diff
@@ -104,71 +104,72 @@ It is also worth having a look at the collections.json file at the root of the f
+   > <br>&nbsp;
 
-This extension allows you to do all these steps in 1 or follow the package, install and instantiate in separate steps. We will use the 1 step method as this process is covered in depth in the Introductory series.
+<br/>
 
-1. Navigate to the `FABRIC ENVIRONMENTS` panel and select the network you created, so if you called it "newNetwork", select  `newNetwork o (click to start)` – this will start your local fabric environment. This may take a few minutes.
-2. Once you've received a notification saying "Connected to newNetwork", click `+ Instantiate` under `Smart Contracts > Instantiated`.
-3. Firstly, you will be asked which peers you would like to instantiate the contract on, select both peers.
-4. Then, select the smart contract you wish to instantiate onto the peers. Select the `privateContract` (or whichever contract name you used in Step 1!).
-5. If you would like to call a function when instantiating, please enter it now, but this is not necessary for this tutorial – so feel free to press enter to move onto the next step.
-6. You'll be asked if you would like to provide a Private data configuration file, click `Yes` and add in your
-`collections.json` file from your `privateContract` folder. This `collections.json` file was created when you created your `Private Data Smart Contract`.
-7. When asked for a smart contract endorsement policy, select `Default (single endorser, any org)`, unless you would like to specify an endorsement policy.
-8. If you are using Typescript, you will `see privateContract@0.0.1` appear in the `SMART CONTRACTS` panel and under `Smart Contracts > Installed` in the `FABRIC ENVIRONMENTS`. Your project has been packaged using the information in `package.json` and installed on all peers in the network.
+**Steps to package and deploy a Private Data Smart Contract:**
+*For more information on packaging and deploying a contract, view tutorial 3 of the basic tutorials series.*
+1. Move the mouse over the title bar of the Smart Contracts view, click the "..." that appears and select "Package Open Project".
+2. Select "tar.gz (V2 channel capabilities)" to deploy this contract in future to a channel with V2 channel capabilities. The Smart Contracts view will be updated to show the new package.
+3. In the Fabric Environments view, click "mychannel" -> "+ Deploy smart contract".
+4. In the Deploy Smart Contract form, select "privateContract@0.0.1" from the drop down list, and click 'Next'.
+5. In step 2 of the form, default values for Definition name and version are provided and don't enter anything for Endorsement policy or Collections configuration as we will be using the preset implicit collections to demonstrate private contracts. Then click 'Next' to move to Step 3 of the deploy. - *For TypeScript smart contracts, both the name and version are taken from the <i>package.json</i> file in the root of the smart contract project.*
```

Comment:
`default values for Definition name and version are provided and don't enter anything for Endorsement policy or Collections configuration as we will be using the preset implicit collections to demonstrate private contracts.`
->
`default values for the definition name and version are provided and don't require any changes. The endorsement policy and collections configuration can be left blank as we will be using the preset implicit collections to demonstrate private contracts.`